docs/source/contributing/test.md

```diff
@@ -4,8 +4,7 @@ In DeepReg, we use [pytest](https://docs.pytest.org/en/stable/) (not
 [unittest](https://docs.python.org/3/library/unittest.html)) for unit tests to ensure a
 certain code quality and to facilitate the code maintenance.
 
-The testing is checked via [Travis-CI](https://travis-ci.com/github/DeepRegNet/DeepReg)
```

Comment:
We should still indicate *where* testing is done; would be best to indicate that a GitHub action/workflow is used for this (and perhaps a link to relevant information).

Reply:
ok